front/lib/swr/vaults.ts

```diff
@@ -280,8 +302,6 @@ export function useUpdateFolder({
       }
     );
     if (res.ok) {
-      void mutateVaultDataSourceViews();
```

Comment:
Any reason we're getting rid of the mutate here?

Reply:
yes, it's useless - we're updating the description, and description is not used in the views list ( I rather call mutate of the single view query in VaultFolderModal to correctly relaod description after update)